.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "themes/docsy"]
 	path = themes/docsy
 	url = https://github.com/google/docsy.git
-	branch = v0.2.0
+	branch = v0.7.2
```

Comment:
IMO, we should bump the minor version one by one and try to review most of the pages to ensure the theme, stylings won't get broken accidentally instead of ignoring over the minor versions.

Some of the CSS styling tied with the theme if I am remembering correctly, direct upgrade will introduce more and more styling issues.

@sftim What is your opinion?

Reply:
I guess we can just review the changes in here, https://github.com/google/docsy/releases and call it.  (also there is some preview deployments that we can do the preview checks as well??)

IMO, there is no major version change, we can probably just move straight to the latest.

Reply:
> IMO, we should bump the minor version one by one and try to review most of the pages to ensure the theme, stylings won't get broken accidentally instead of ignoring over the minor versions.

Yes, I'd do that.

> IMO, there is no major version change, we can probably just move straight to the latest.

I'd support that if the major version was greater than or equal to 1. But, it isn't.